dev/docs/best_practices/local-observability.md

```
@@ -93,21 +130,17 @@ LW_OBS_GRAFANA_PORT=3100 LW_OBS_OTLP_HTTP_PORT=4319 make observability
 
-Three ways, all wired by `make observability-connect`:
+Two ways, both wired by `make observability-connect`:
 
-1. **Grafana MCP** — a `grafana-local` MCP server (the same `grafana/mcp-grafana`
-   image used for `grafana-prod`) pointed at the local Grafana with a minted
-   service-account token. Restart Claude Code to load it, then the
-   `query_loki_logs` / `query_prometheus` / Tempo tools work against local data.
-2. **gcx CLI** — Grafana's official CLI. `connect` runs `gcx login local
+1. **gcx CLI** — Grafana's official CLI. `connect` runs `gcx login local
    --server http://localhost:3000 --token <token>`. Then:
    ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)